src/components/layout/BasicLayout.jsx

```diff
@@ -9,56 +9,115 @@ import { connect } from 'react-redux';
 import { selectors } from '../../store';
 import '../../../assets/css/basicLayout.css';
 
-const BasicLayout = props => {
+class BasicLayout extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { width: 0 };
+    this.updateWindowDimensions = this.updateWindowDimensions.bind(this);
+    this.layoutToDisplay = this.layoutToDisplay.bind(this);
+  }
+  
+  componentDidMount() {
+    this.updateWindowDimensions();
+    window.addEventListener('resize', this.updateWindowDimensions);
+  }
+  
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.updateWindowDimensions);
+  }
+  
+  updateWindowDimensions() {
+    this.setState({ width: window.innerWidth });
+  }
 
-  const contentCols = props.patient ? 10 : 12;
-
-
-  const colWidth = props.patient ? '55%' : '100%';
-
-  return (
-    <div id="outer-container" className="ag-theme-material">
-      <ReduxToastr
-        closeOnToastrClick
-        timeOut={1000}
-      />
-      <Grid fluid={true}>
-        <Row className="header-alt-row">
-          <HeaderAlt
-            className="HeaderAlt"
-            logo={props.logo}
-            navMenuPages={props.navMenuPages}
-            userMenuPages={props.userMenuPages}
-          />
-        </Row>
-        <Row>
-          {props.patient &&
-          <PatientHeader
-            identifierTypesToDisplay={props.identifierTypesToDisplay}
-            patient={props.patient}/>
-          }
-        </Row>
-        <Row className="basic-layout">
-          {props.patient && props.leftRail &&
+  layoutToDisplay(contentCols, colWidth) {
+    if (this.state.width < 600) {
+      if (this.props.path === '/screening') {
```

Comment:
@jomadoye I like the responsiveness here, but the issue is (and it may be tricky to resolve) is that we can't depend on checking that props path is "screening" since that path is specific to the Workflow/Screening app and we'd like to use this with multiple apps...

Let me talk with Dave and Mike when they are out of a meeting, but I'm thinking we might just abandon (for now) the side-by-side layout and entirely, I will follow up with another ticket...